Formula/hh-suite.rb

```diff
@@ -31,9 +30,16 @@ def install
     end
   end
 
+  def caveats
+    "MMseqs2 requires at least SSE4.1 CPU instruction support." unless Hardware::CPU.sse4?
+  end
+
   test do
-    assert_match "Usage", shell_output("#{bin}/hhalign -h")
-    assert_match "Usage", shell_output("#{bin}/hhblits -h")
-    assert_match "Usage", shell_output("#{bin}/hhsearch -h")
+    assert_match "HHalign 3.1.0", shell_output("#{bin}/hhalign -h | head -1")
+    assert_match "HHblits 3.1.0", shell_output("#{bin}/hhblits -h | head -1")
+    assert_match "HHconsensus 3.1.0", shell_output("#{bin}/hhconsensus -h | head -1")
+    assert_match "HHfilter 3.1.0", shell_output("#{bin}/hhfilter -h | head -1")
+    assert_match "HHmake 3.1.0", shell_output("#{bin}/hhmake -h | head -1")
+    assert_match "HHsearch 3.1.0", shell_output("#{bin}/hhsearch -h | head -1")
```

Comment:
I prefer not to hard code a version number in the `test do` block. It should not be necessary to update the `test` block with each new version. Checking for `Usage` is good enough.
The `| head -n1` is unnecessary.